### PR TITLE
zoomLimits parameter for KML

### PR DESCRIPTION
**Pull request by danji90**
# How to

A _zoomLimits_ parameter has been added to the KML read/write methods. It writes a stringified array into the kml and parses it into the feature properties when reading.

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

### CODE REVIEW

File: src/utils/KML.js [hunk 1]
```diff
@@ -433,6 +436,11 @@ const writeFeatures = (layer, featureProjection, mapResolution) => {
       newStyle.fill = null;
     }
 
+    // In case a fill pattern should be applied (use fillPattern attribute to store pattern id, color etc)
```
Comment (oterral) [inline src/utils/KML.js:439]: Is this comment reöltaed to zoomLimits?

**Comment by danji90** [comment]
@oterral I fixed the comment

File: src/utils/KML.js [hunk 2]
```diff
@@ -55,11 +55,21 @@ const getLineIcon = (feature, icon, color, start = true) => {
...
   let styles = feature.getStyleFunction();
 
+  // Store maxZoom in properties
+  if (feature.get('maxZoom')) {
+    feature.set('maxZoom', parseInt(feature.get('maxZoom'), 10));
+  }
+
+  // Store minZoom in properties
+  if (feature.get('minZoom')) {
+    feature.set('minZoom', parseInt(feature.get('minZoom'), 10));
```
Comment (oterral) [inline src/utils/KML.js:70]: I would use parseFloat maybe we will want to use 8.5 some day

**Review by oterral — CHANGES_REQUESTED** [review]
Use parseFloat instead of parseInt

**Pull request merged**